package.json

```diff
@@ -21,10 +21,10 @@
     "@types/mysql": "^0.0.31",
     "@types/pg": "^7.4.14",
     "chai": "^3.5.0",
-    "mocha": "^3.2.0",
-    "mysql": "^2.17.1",
+    "mocha": "^6.1.4",
```

Comment:
When I tried upgrading mocha, the test just hung. did you test to make sure it does not hang?

Reply:
Yes, seeing some weird errors, will look into it.

Reply:
@reynardmh Ok, should be fixed now! I also updated the `Readme.md`.